jta-1_2-in-wildfly/README.md

```diff
@@ -1,26 +1,31 @@
 JTA 1.2 in Wildfly example.
-==================================================================================================
-Author: Gytis Trikleris;
-Level: Intermediate;
+===
+
+Author: Gytis Trikleris<br/>
+Level: Intermediate<br/>
 Technologies: JTA, JPA, JMS
 
 What is it?
------------
+---
 
 This example demonstrates how to use new JTA 1.2 features inside WildFly application server.
 
 
-QuickstartEntityRepository and QuickstartQueue classes demonstrace usage of @Transactional annotation.
-TransactionScopedPojo class demonstrates @TransactionScoped annotation. TestCase class is used to drive the example.
+`QuickstartEntityRepository` and `QuickstartQueue` classes demonstrace usage of `@Transactional` annotation.
+`TransactionScopedPojo` class demonstrates `@TransactionScoped` annotation. `TestCase` class is used to drive the example.
 
 
 Build and Deploy the Quickstart
--------------------------------
+---
 
 In order to run quickstart in the managed Wildfly application server, run the following command:
 
-        mvn clean install
+```
+mvn clean install
```

Comment:
@ochaloup this and related was what I meant when talking about the "code changes", sorry I was not clear

Reply:
I see you resolved that though - thanks!